MIP84/MIP84.md

```diff
@@ -80,61 +88,128 @@ This MIP will activate the Protocol-Owned Vault Emulation (POVE). It is an appro
+
+This subproposal process enables Maker Governance to modify the Emulated Surplus Buffer Upper Limit as defined in MIP84c9
+
+#### Subproposal Parameters and Template
+
+The parameters for MIP84c10 subproposals are:
+
+- **Minimum Feedback Period**: 4 weeks.
+- **Minimum Frozen Period**: 1 week.
+- **Governance Cycle**: Monthly.
+
+Facilitators with relevant mandates may propose weekly polls to amend the Protocol Surplus and the Emulated Surplus Buffer Upper Limit when they deem necessary.
+
+MIP84c10 subproposals must use the template located at [MIP84c10-Subproposal-Template.md](https://github.com/makerdao/mips/blob/master/MIP84/MIP84c10-Subproposal-Template.md).
+
+### MIP84c11: Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling
+
+To prevent overleverage of the POVE due to unsustainable price appreciation of the POVE's collateral assets, this MIP defines virtual price ceilings for POVEC. If the price of the collateral assets is above the POVEC virtual price ceilings, they are only counted as being equal to the virtual price ceiling when calculating the POVEDC.
+
+The POVEC Virtual Price Ceiling of ETH (For Lido Staked ETH or EtherDai): 1500 USD
+
+
+### MIP84c12: Modify Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling
+
+This component enables a Subproposal Process to modify the POVEC Virtual Price Ceiling defined in MIP84c11.
+
+#### Subproposal Parameters and Template
 
-The initial Emulated Surplus Buffer Upper Limit is set to 20 million Dai.
+The parameters for MIP84c12 subproposals are:
 
-### MIP84c10: Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling
+- **Minimum Feedback Period**: 4 weeks.
+- **Minimum Frozen Period**: 1 week.
+- **Governance Cycle**: Monthly.
 
-To prevent overleverage of the POVE due to unsustainable price appreciation of the POVE's collateral assets, this MIP defines virtual price ceilings for POVEC. If the price of the collateral assets are above the POVEC virtual price ceilings, they are only counted as being equal to the virtual price ceilings when calculating the POVEDC.
+Facilitators with relevant mandates may propose weekly polls to amend the Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling when they deem necessary.
 
-The POVEC Virtual Price Ceiling of ETH (For Lido Staked ETH or EtherDai): 2000 USD
+MIP84c12 subproposals must use the template located at [MIP84c12-Subproposal-Template.md](https://github.com/makerdao/mips/blob/master/MIP84/MIP84c12-Subproposal-Template.md).
 
-The POVEC Virtual Price Ceiling of LDO: 3 USD
+# MIP84c13: Long-term replacement of the Protocol-Owned Vault Emulation
 
-### MIP84c11: Modify Protocol-Owned Vault Emulation Collateral Virtual Price Ceiling
+The POVE is not meant to be a permanent feature of the Endgame, rather it is a short-term solution that allows Maker to quickly begin accumulating staked ETH, which enables it to be done at a slower rate, reducing risks.
```

Comment:
```suggestion
The POVE is not meant to be a permanent feature of the Endgame; rather, it is a short-term solution that allows Maker to quickly begin accumulating staked ETH at a slower rate, reducing risks.
```

Reply:
Not 100% sure about this one

Reply:
He's saying it's quicker than a completely automated solution but it doesn't make a whole lot of sense as written. How about the below?

```suggestion
The POVE is not meant to be a permanent feature of the Endgame, rather it is a short-term solution that allows Maker to quickly begin accumulating staked ETH at a steady rate, reducing risks.
```